flashnn/triton_kernels/gemm_a8w8.py

```diff
@@ -186,17 +208,14 @@ def triton_gemm_a8w8_forward(out, a, b, alpha_row, alpha_col):
         K,
         a.stride(0),
         a.stride(1),
-        b.stride(0),
         b.stride(1),
+        b.stride(0),
```

Comment:
It seems that b.stride(1) is b_stride_k?
So b.shape = n, k?

Reply:
Yes, b is created as shape (n, k).

Reply:
why don't you do a b = b.T? I know this is not necessary. I just found it a little confused to see 
b.stride(1) is passed before b.stride(0)

Reply:
I revert the signature change in my PR, now we can follow `b.stride(0)` before `b.stride(1)`